pilot/pkg/bootstrap/namespacecontroller.go

```diff
@@ -51,83 +51,77 @@ type NamespaceController struct {
 	getData func() map[string]string
 	client  corev1.CoreV1Interface
 
-	queue queue.Instance
-
 	// Controller and store for namespace objects
 	namespaceController cache.Controller
-	// Controller and store for ConfigMap objects
-	configMapController cache.Controller
+	namespaceStore      cache.Store
 }
 
 // NewNamespaceController returns a pointer to a newly constructed NamespaceController instance.
 func NewNamespaceController(data func() map[string]string, kubeClient kubernetes.Interface) *NamespaceController {
 	c := &NamespaceController{
 		getData: data,
 		client:  kubeClient.CoreV1(),
-		queue:   queue.NewQueue(time.Second),
```

Comment:
why are we removing the queue? This is standard across istio controllers to properly manage error handling, etc

Reply:
The queue is removed to simplify the implementation. For example, the service controller in security/pkg/registry/kube/service.go and the service account controller in security/pkg/registry/kube/serviceaccount.go also do not use a queue.

Reply:
The queue was added to rate limit change and allow work to be easily retried for transient failures. This is a common k8s controller pattern. See https://github.com/kubernetes/sample-controller/blob/master/controller.go